app/controllers/users/reset_passwords_controller.rb

```diff
@@ -49,10 +49,14 @@ def update
 
     protected
 
+    def email
+      params[:password_reset_email_form][:email]
```

Comment:
No, because this ends up being passed to `User.find_with_email` which already downcases and strips.